lib/attr_encrypted.rb

```diff
@@ -234,9 +234,9 @@ def attr_encrypted?(attribute)
   #     attr_encrypted :email
   #   end
   #
-  #   email = User.decrypt(:email, 'SOME_ENCRYPTED_EMAIL_STRING')
-  def decrypt(attribute, encrypted_value, options = {})
-    options = encrypted_attributes[attribute.to_sym].merge(options)
+  #   email = User.attr_encrypted_decrypt(:email, 'SOME_ENCRYPTED_EMAIL_STRING')
+  def attr_encrypted_decrypt(attribute, encrypted_value, options = {})
```

Comment:
Renames the `.decrypt` method (I don't think this or the next one impact Rails 7)